assets/ajax-cart.js.liquid

```diff
@@ -0,0 +1,627 @@
+/*============================================================================
+  (c) Copyright 2015 Shopify Inc. Author: Carson Shold (@cshold). All Rights Reserved.
+
+  Plugin Documentation - http://shopify.github.io/Timber/#ajax-cart
+
+  Ajax the add to cart experience by revealing it in a side drawer
+
+  This file includes:
+    - Basic Shopify Ajax API calls
+    - Ajax cart plugin
+
+  This requires:
+    - jQuery 1.8+
+    - handlebars.min.js (for cart template)
+    - modernizer.min.js
+    - snippet/ajax-cart-template.liquid
+
+  JQUERY API (c) Copyright 2009-2015 Shopify Inc. Author: Caroline Schnapp. All Rights Reserved.
+  Includes slight modifications to addItemFromForm.
+==============================================================================*/
+if ((typeof Shopify) === 'undefined') { Shopify = {}; }
+
+/*============================================================================
+  Basic JS Helper Functions
+==============================================================================*/
+function urlParams (name) {
+  name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+  var regex = new RegExp("[\\?&]" + name + "=([^&#]*)"),
+      results = regex.exec(location.search);
+  return results == null ? "" : decodeURIComponent(results[1].replace(/\+/g, " "));
+}
+
+/*============================================================================
+  API Helper Functions
```

Comment:
hmm, ok, I guess in the future these decisions need to be weighed as cost vs benefit. Right now it's confusing since it's unclear if you'd ever include the other version of the file and it introduces the possibility of one file overwriting the other. I'd say it probably isn't worth it given that complexity.  It'd be helpful to document the preferred flow here - eg if using Timber, should you never include that api script because its functionality is recreated here? What if you don't have an ajax cart, would you use that script then?